src/app/core/database/pouch-database.ts

```diff
@@ -29,9 +29,9 @@ import { PerformanceAnalysisLogging } from "../../utils/performance-analysis-log
  * should be implemented in the abstract {@link Database}.
  */
 export class PouchDatabase extends Database {
-  static async createWithData(data: any[]): Promise<PouchDatabase> {
+  static createWithData(data: any[]): PouchDatabase {
     const instance = PouchDatabase.createWithInMemoryDB();
-    await Promise.all(data.map((doc) => instance.put(doc)));
+    data.forEach((doc) => instance.put(doc, true));
```

Comment:
The problem is, this function is only used in storybook examples for providing the database. However, in Storybook it is not possible to wait till the promise resolves and prior these changes, some stories were falsely using the return promise in `userValue` (e.g. `{ provide: Database, useValue: PouchDatabase.createWithData([...]) }`). In this example the the `Database` will be a promise rather than the actual database. I couldn't see another way of dealing with this issue.

Regarding the second issue. Yes it is not totally related to this PR but I was checking the Storybook-Stories because many of them were affected by the changes of this PR. Some of the stories I fixed also included adaptions that were not related to changes of this PR. How should this be handled instead?